Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM node:lts-alpine as development
+FROM node:14.18.1-slim as development
```

Comment:
Just wondering whats your rationale is behind pinning this version of node here. The reason why I agree is a good idea is as an API developer, I have set my default node version to 14.18.1 and this will ensure that if Kinetic runs locally for a developer, it will also run on the container.